Notebooks/verify_truth.ipynb

```diff
@@ -0,0 +1,707 @@
+    "print(pq_df.columns)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pq_df.sort_values(by=['bandpass', 'MJD'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Plotting"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_band_lc(times, fluxes, params):\n",
+    "    out = plt.scatter(np.asarray(times), np.asarray(fluxes), **params)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def format_title(id, ra, dec, band=None, object='star'):  \n",
+    "    if band is None:\n",
+    "        return f'Per-band light curves for star {id} at (ra,dec)=({ra}, {dec})'\n",
+    "    else:\n",
+    "        return f'Light curve for {object} {id}, band={band} at (ra,dec)=({ra}, {dec})'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_object(title, the_data, data_source, band):\n",
+    "    '''\n",
+    "    Plot requested band\n",
+    "    \n",
+    "    Parameters\n",
+    "    -----------\n",
+    "    title : string\n",
+    "    the_data : list of data frame which must include columns filtername, obsstart, mag\n",
+    "    data_source : list of data source names\n",
+    "    band : Must be one of ['r', 'i','g', 'u', 'y', 'z']\n",
+    "    '''\n",
+    "    if band not in ['r', 'i','g', 'u', 'y', 'z']:\n",
+    "        print('Unknown band \"',band,'\"')\n",
+    "        return\n",
+    "    color_dict  = {'r' : 'red', 'g' : 'green', 'i' : 'orange', 'u' : 'magenta',\n",
+    "                  'y' : 'blue', 'z' : 'black'}\n",
+    "    \n",
+    "    plt.figure(figsize=(18, 6))\n",
+    "    plt.suptitle(title)\n",
+    "    plt_shape_min = 100 + 10 * len(data_source) + 1\n",
+    "    for i in range(len(data_source)):\n",
+    "        plt.subplot(plt_shape_min + i).set_title(data_source[i])\n",
+    "    \n",
+    "        good_d = the_data[i][(np.isnan(the_data[i].delta_flux)==False)]\n",
+    "        band_d = good_d[(good_d.bandpass==band)]\n",
+    "        plt.xlabel('Julian date')\n",
+    "        plt.ylabel('Delta flux')\n",
+    "        params = {'marker' : 'o', 'label' : f'band {band}', 'color' : color_dict[band],\n",
+    "                 's' : 1.0}\n",
+    "\n",
+    "        plot_band_lc(list(band_d['MJD']), list(band_d['delta_flux']),\n",
+    "                    params)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for band in ('i', 'r', 'g', 'u'):\n",
+    "    title = format_title(id, lc_ra, lc_dec, band, object='agn')\n",
+    "    plot_object(title, [pg_df, sq_df, pq_df], ('PostgreSQL', 'SQLite', 'Parquet'), band)"
+   ]
+  },
```

Comment:
```suggestion
```

I think this cell should be removed? Since `lc_ra`, `lc_dec` etc are no longer defined in the notebook?

Reply:
I do intend to make those plots.  The mistake was to not save the values I needed from an earlier query. That's fixed now.  I also found some other minor things to clean up.